### PR TITLE
UG-260 Reword graduation date

### PR DESCRIPTION
**Pull request by bolajiadeyemi**
### Description
Reword graduation date field label and add compliance text. 

### Ticket
https://financialtimes.atlassian.net/browse/UG-260

### Screenshots

| Before | After |
| ------ | ----- |
|     
Screenshot 2020-09-30 at 10 55 16
   |    
Screenshot 2020-09-30 at 10 54 40
   |

### Reminder
Have you completed these common tasks (remove those that don't apply)?

- [x] **Documentation** updated or created
- [x] **Tests** written for new or updated for existing functionality
- [x] **Stories** updated to use this change
- [x] **Accessibility** checked for screen readers and contrast
- [x] **Design Review** ran past the designer
- [x] **Product Review** ran past the product owner

### CODE REVIEW

File: components/graduation-date.jsx [hunk 1]
```diff
@@ -80,6 +88,7 @@ export function GraduationDate ({
 					Please select a valid graduation date
 				</span>
 			</div>
+			<Compliance/>
```
Comment (nickramsbottom) [inline components/graduation-date.jsx:91]:
```suggestion
			<Compliance />
```

**Review by nickramsbottom — CHANGES_REQUESTED** [review]
tiny change, looks good other than that!

**Review by nickramsbottom — APPROVED** [review]
nice!

**Comment by bolajiadeyemi** [comment]
> nice!

Thank you

**Pull request merged**